pyproject.toml

```diff
@@ -1,3 +1,103 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+  "pytest-runner",
+  "setuptools>=61.2",
+]
+
+[project]
+name = "kornia"
+version = "0.6.11.dev"
+description = "Open Source Differentiable Computer Vision Library for PyTorch"
+[project.readme]
+file = "README.md"
+content-type = "text/markdown"
```

Comment:
i think is kinda hard to read `toml`, maybe using tabs for the same top-level looks better? 

something like this
```toml
[project]
    name = "kornia"
    version = "0.6.11.dev"
    description = "Open Source Differentiable Computer Vision Library for PyTorch"
     
    [project.readme]
        file = "README.md"
        content-type = "text/markdown"
```

Reply:
cclauss
> Should tab characters be used in toml files?

ChatGPT:
> No, it is recommended to use only spaces and not tab characters in TOML files.

> According to the TOML specification, indentation must be done using only spaces, and not tabs. In fact, the specification states that "TAB characters must not appear anywhere in the file". This is because different editors and systems can interpret tabs differently, which can cause issues with parsing the TOML file consistently across different environments.

> To ensure that your TOML file is properly formatted and can be parsed consistently across different systems, it is recommended to use only spaces for indentation. The convention is to use a width of 4 spaces per level of indentation, although this can vary depending on your personal preference or the conventions of your team or organization.

Reply:
4 spaces instead of the tab them? 😅

Reply:
[`pyproject-fmt --indent 4 pyproject.toml`](https://pyproject-fmt.readthedocs.io/en/latest/#pyproject-fmt-options)